enable/tests/qt4/mouse_wheel_test_case.py

```diff
@@ -43,31 +43,55 @@ def setUp(self):
     def test_vertical_mouse_wheel(self):
         from pyface.qt import QtCore, QtGui
 
+        is_qt4 = (QtCore.__version_info__[0] <= 4)
+
         # create and mock a mouse wheel event
-        qt_event = QtGui.QWheelEvent(
-            QtCore.QPoint(0, 0), 200, QtCore.Qt.NoButton, QtCore.Qt.NoModifier,
-            QtCore.Qt.Vertical
-        )
+        if is_qt4:
+            qt_event = QtGui.QWheelEvent(
+                QtCore.QPoint(0, 0), 200, QtCore.Qt.NoButton,
+                QtCore.Qt.NoModifier, QtCore.Qt.Vertical
+            )
+        else:
+            qt_event = QtGui.QWheelEvent(
+                QtCore.QPointF(0, 0),
+                self.window.control.mapToGlobal(QtCore.QPoint(0, 0)),
+                QtCore.QPoint(0, 200), QtCore.QPoint(0, 200.0/120), 200,
+                QtCore.Qt.Vertical, QtCore.Qt.NoButton, QtCore.Qt.NoModifier,
+                QtCore.Qt.ScrollUpdate
+            )
```

Comment:
I feel like this code should be in a pyface testing support module... But I don't see any such module in pyface.

Reply:
Pyface doesn't really do any testing at the level of individual mouse events (things are usually driven at the level of signals and slots), so we haven't had a need to put anything like this into Pyface.

For now it's probably good enough to have it as part of `enable.tests.qt4`; if we find that we are repeating this a lot we might want to create a `enable.qt4.testing.*` utility method.

Reply:
Sounds good to me